apps/web-staking/src/app/components/redeem/History.tsx

```diff
@@ -305,22 +304,27 @@ export default function History({ redemptions, reloadRedemptions }: {
 						})}
 					</div>
 				}
+				{redemptionsLoading && <div className="bg-nulnOil/75 shadow-default mb-[53px]">
+					<MainTitle isSubHeader classNames="!text-3xl capitalize border-b-1 border-chromaphobicBlack py-6 md:px-8 px-[17px] !mb-0" title="Loading Redemptions..." />
+				</div>
+					
+				}
```

Comment:
Just for readability I would think this should be above the redemption mappings, what do you think ?

Reply:
Makes sense to me, it has been moved!